src/api/app/views/webui/feeds/notifications.rss.builder

```diff
@@ -9,13 +9,12 @@ xml.rss version: '2.0' do
 
     @notifications.each do |notification|
       xml.item do
-        xml.title notification.event.subject
-        # We render the event_mailer view to 'description'
+        xml.title notification.title
         xml.description render(
-          template: "event_mailer/#{notification.event.template_name}",
+          template: "notifications/#{notification.template_name}",
```

Comment:
Hm, do you ever expect those to be different from the mailer content? Why duplicate all of this?

Reply:
They will be not so different. At least by now the only difference is that the diff of a submit request is not show in the body of a RSS feed item.

Another reason to duplicate templates was not to touch mailer templates. Using the same templates for both mails and RSS feed items would pretty probably mean to change variables in the templates, and create some logic to fill this variables with different objects, depending on rendering mails for an event, or rendering a RSS feed item from a notification.

What do you think?

Reply:
Hm maybe I don't understand something. The RSS feeds now use the mailer templates right? Why change this now?

Reply:
> Hm maybe I don't understand something. The RSS feeds now use the mailer templates right?

Yes.

> Why change this now?

This has been changed to use other templates to not touch the mailer templates, and to substitute the variables inside the templates with the values retrieved from the new polymorphic columns from notifications.

Reply:
Ah, you are getting the data from other places. I understand. Okay